diagnostic.sql

```diff
@@ -5,24 +5,43 @@ CREATE DATABASE "sql-crud-assessment";
 
 -- Question 1. Create a table called `licensees` to hold data in columns surname, given_name, gender, height, weight, born_on, eye_color, hair_color.  See `data/licensees.csv` for example data.  Don't forget that all tables should have an id column as a unique identifier for a row.
 
--- your code here
+CREATE TABLE licensees (
+  id SERIAL PRIMARY KEY,
+  surname TEXT,
+  given_name TEXT,
+  gender TEXT,
+  height INTEGER,
+  weight INTEGER,
+  born_on DATE,
+  eye_color TEXT,
+  hair_color TEXT
+);
 
 -- Question 2. Add `Bishop, Caren, f, 63, 132, 1943-09-26, Brown, Black` into the table of licensees.
 
--- your code here
+INSERT INTO licensees (surname,given_name,gender,height,weight,born_on,eye_color,hair_color)
+VALUES ('Biship', 'Caren', 'f',63,132,'1943-09-26', 'Brown', 'Black');
 
 -- After completing questions 1 and 2 uncomment the following to load data from `data/licensees.csv` into the `licensees` table.
 
--- \copy licensees(surname,given_name,gender,height,weight,born_on,eye_color,hair_color) FROM './data/licensees.csv' WITH (FORMAT csv, HEADER true)
+\copy licensees(surname,given_name,gender,height,weight,born_on,eye_color,hair_color) FROM './data/licensees.csv' WITH (FORMAT csv, HEADER true)
 
 -- Question 3. Write a query to get all attributes of licensees with Hazel eye color and Brown or Black hair color.
 
--- your code here
+SELECT * eye_color
+FROM licensees
+WHERE eye_color = 'Hazel' AND hair_color = 'Brown' OR HAIR_color = 'Black'
```

Comment:
The WHERE clause is also not making the right query. This is because, according to [Operator Precedence](http://www.postgresql.org/docs/7.2/static/sql-precedence.html), AND operations are always run before OR operations, so it would be as if we'd written `WHERE (eye_color = "Hazel" AND hair_color = "Brown") OR hair_color =  "Black"`; as a result we might accidentally select someone with black hair but _without_ hazel eyes.